### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by stugerb** [flippelt/WnG-Apocrypha-FoundryVTT#13] [General]
Wanted to say thank you for this. Makes it so much easier!
Not meaning to be pushy, just asking, do you plan on adding the other Abundance of Apocrypha items or just those for PCs?
Like the ones for specific threats?
And I am guessing that you are waiting for Church of Steel before doing the vehicles?
I am dyslexic and have tried taking coding classes but it is incredibly difficult for me, so I am really grateful for all that you have already accomplished.
Best wishes!

**Comment by flippelt** [comment]
Hello,

As someone who's also dyslexic and have ADHD I totally understand your
problem.

About the items for threats and the vehicles.
In the ideal world, yes, I was waiting for it to be available within the
system to do vehicles and I would also like to make new modules for the
Threats. But since I'm in a new job that is pretty much sucking all the
life and free time I have, I can't give you any prevision for when I'm
gonna start work on those.
The last update was already something I didn't have time to do and a good
member of the Community did it all getting from where I stopped.
There's still things to finish like Archetypes and Ascension Packages and
until those are completed I'll not start any new projects, specially the
one of the Apocrypha Beastiaries.

Felipe Lippelt
*Journalist*


Em sáb., 20 de abr. de 2024 às 11:33, stugerb ***@***.***>
escreveu:
